SwrveSDK/src/main/java/com/swrve/sdk/localstorage/SQLiteLocalStorage.java

```diff
@@ -124,6 +124,7 @@ public void setSecureCacheEntryForUser(String userId, String category, String ra
 
     private void insertOrUpdate(String table, ContentValues values, String whereClause, String[] whereArgs) {
         if (connectionOpen.get()) {
+            SwrveLogger.i("SQLLite", "Insert/Open into " + database + " isDbLockedByCurrentThread: " + database.isDbLockedByCurrentThread() + " isOpen: " + database.isOpen());
```

Comment:
I see these guys as temp that will be removed after we get a more reasonable console from Jenkins.